### ISSUE TITLE
ForceFeedback.h: No such file or directory - Trouble setting up project on ROS1 branch

### ISSUE THREAD

**Issue by Vandor1** [kuriatsu/ros-g29-force-feedback#29]
I am on Ubuntu 18.04 with ROS1 and following your readme in that branch I get the following error when I run catkin build.

Running a clean install and running catkin_make gives the same error.

catkin_ws/src/ros-g29-force-feedback/src/g29_force_feedback.cpp:8:10: fatal error: ros_g29_force_feedback/ForceFeedback.h: No such file or directory
 #include "ros_g29_force_feedback/ForceFeedback.h"

**Comment by kuriatsu** [comment]
Are there any other errors?
And please try
```bash
source catkin_ws/install/setup.bash
rosmsg show ros_g29_force_feedback/ForceFeedback
```
to check whether the message is correctly built.

**Comment by Vandor1** [comment]
I solved my issue by running: pip install empy==3.3.4.

**Comment by kuriatsu** [comment]
That's strange LOL.
